source/request.ts

```diff
@@ -48,6 +48,9 @@ export function prepareRequestOptions(
     if (context.maxBodyLength) {
         finalOptions.maxBodyLength = context.maxBodyLength;
     }
+    if (userOptions.hasOwnProperty('onUploadProgress')) {
+        finalOptions.onUploadProgress = userOptions['onUploadProgress'];
```

Comment:
Could you please change this to use double quotes?

Reply:
Or better yet, add this as an optional property to user options and just use `if (userOptions.onUploadProgress)` etc. :)